### PR TITLE
[Regression] Fix a critical error when importing ove files

### PR DESCRIPTION
**Pull request by Howard-C**
Recently introduced in https://github.com/musescore/MuseScore/commit/b1ab9574cc52177b14d9e6f49c4bd2fbfdc246af.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
b1ab9574

File: mscore/ove.cpp [hunk 1]
```diff
@@ -5183,7 +5183,7 @@ int getInt(int byte, int bits) {
       int num = 0;
 
       if( bits > 0 ) {
-            int factor = int(pow(2.0, bits-1));
+            int factor = 1 << (bits - 1);
```
Comment (anatoly-os) [inline mscore/ove.cpp:5186]: Could you please leave a comment that this operation equals to the `pow(2,bits-1)`? Same above

Comment (anatoly-os) [reply mscore/ove.cpp:5186]: And force push the changes! Let's test the fix proposed by the Travis team and applied by me 2 mins ago :)

Comment (Jojo-Schmitz) [reply mscore/ove.cpp:5186]: Is it really? What if bits is 2? 2-1 is 1, square is still one, but shift 1 is 2

Comment (Howard-C) [reply mscore/ove.cpp:5186]: `pow(2, 1)` is of course 2...

Comment (Howard-C) [reply mscore/ove.cpp:5188]: @anatoly-os Done!

**Comment by anatoly-os** [comment]
Nope, Travis doesn't work :(

**Comment by Howard-C** [comment]
Still, doesn't go in the way of merging this PR ;-)

**Comment by AntonioBL** [comment]
Travis has magically reappeared! 😃

**Pull request merged**